.travis.yml

```diff
@@ -1,9 +1,27 @@
 language: objective-c
-osx_image: xcode6.4
+osx_image: xcode7.2
+sudo: false
```

Comment:
So it uses the new container-based infrastructure: https://docs.travis-ci.com/user/migrating-from-legacy/